src/app/calendar/ParcelCalendar.tsx

```diff
@@ -13,20 +26,67 @@ const Centerer = styled.div`
 
 const CalendarWrapper = styled.div`
     width: 100vmin;
-    margin: 2em;
+    margin: 0 2em 2em;
 `;
 
-interface ParcelCalendarProps {
-    events: CalendarEvent[];
-}
-
 const ParcelCalendar: React.FC<ParcelCalendarProps> = (props) => {
+    const chosenTheme = useContext(ChosenThemeContext);
+
+    const colorMap: LocationColorMap = {
+        "Brixton Hill - Methodist Church": {
+            color: `${chosenTheme.rainbow.red.background[0]}`,
+            text: `${chosenTheme.rainbow.red.foreground[0]}`,
+        },
+        "Clapham - St Stephens Church": {
+            color: `${chosenTheme.rainbow.orange.background[0]}`,
+            text: `${chosenTheme.rainbow.orange.foreground[0]}`,
+        },
+        "N&B - Emmanuel Church": {
+            color: `${chosenTheme.rainbow.yellow.background[0]}`,
+            text: `${chosenTheme.rainbow.yellow.foreground[0]}`,
+        },
+        "Streatham - Immanuel & St Andrew": {
+            color: `${chosenTheme.rainbow.lightGreen.background[0]}`,
+            text: `${chosenTheme.rainbow.lightGreen.foreground[0]}`,
+        },
+        "Vauxhall Hope Church": {
+            color: `${chosenTheme.rainbow.darkGreen.background[0]}`,
+            text: `${chosenTheme.rainbow.darkGreen.foreground[0]}`,
+        },
+        "Waterloo - Oasis": {
+            color: `${chosenTheme.rainbow.blue.background[0]}`,
+            text: `${chosenTheme.rainbow.blue.foreground[0]}`,
+        },
+        "Waterloo - St George the Martyr": {
+            color: `${chosenTheme.rainbow.blue.background[1]}`,
+            text: `${chosenTheme.rainbow.blue.foreground[1]}`,
+        },
+        "Waterloo - St Johns": {
+            color: `${chosenTheme.rainbow.purple.background[0]}`,
+            text: `${chosenTheme.rainbow.purple.foreground[0]}`,
+        },
+        Delivery: {
+            color: `${chosenTheme.rainbow.purple.background[1]}`,
+            text: `${chosenTheme.rainbow.purple.foreground[1]}`,
```

Comment:
I just want to standardize so each rainbow color has three gradients
I don't know what will be the other applications for rainbow color, but at least in both here and the data viewer needs at least 2, so I will probably just put here first because it will be hard to further open another theming ticket if more color is needed